server/eclipse-project/src/main/java/us/freeandfair/corla/csv/DominionCVRExportParser.java

```diff
@@ -361,17 +382,17 @@ public synchronized boolean parse() {
       // add the contests to the county we're working with  
       my_county.contests().addAll(my_contests);
       Persistence.saveOrUpdate(my_county);
-    } else {
-      abort();
     }
+    
     return result;
   }
-
+  
```

Comment:
Added new stray whitespace. Not worth a commit, but we should really trim trailing whitespace on all the docs and Java files.